src/setupTests.ts

```diff
@@ -3,3 +3,7 @@
 // expect(element).toHaveTextContent(/react/i)
```

Comment:
Should be removed?

Reply:
I'd rather keep it there. It's a comment added by create-react-app.
I think its more clear to leave it in :-)